rlmolecule/molecule/policy/preprocessor.py

```diff
@@ -53,8 +53,14 @@ class MolPreprocessor(nfp.preprocessing.SmilesPreprocessor):
     def padded_shapes(self, *args, **kwargs):
         return filter_keys(super().padded_shapes(*args, **kwargs))
 
-    def construct_feature_matrices(self, mol: rdkit.Chem.Mol, train: bool = False) -> {}:
-        """ Convert an rdkit Mol to a list of tensors
+    def construct_feature_matrices(self,
+                                   mol: rdkit.Chem.Mol,
+                                   train: bool = False,
+                                   max_num_atoms: Optional[int] = None,
+                                   max_num_bonds: Optional[int] = None,
```

Comment:
we should raise an error if the `max_num_*` arguments are exceeded by the molecules

Reply:
This code has been moved to nfp.